Changes

```diff
@@ -1,5 +1,25 @@
 Release history for Zonemaster component Zonemaster-GUI
 
+v4.1.0 2023-06-21 (public release version)
+
+ [Features]
+ - Gives small improvement of the run test page (#416)
+ - Makes room for optional footer logo (#408)
+
+ [Fixes]
+ - Updates translations (#418, #419, #423, #422, #420, #406,
+   #405)
```

Comment:
Is there a logic in the PR reference order?

Reply:
Not really. I just take them in the order that I find them. Usually from to to bottom, but sometimes I merge several into one. Do you think the disorder is bad?